app/screens/global_drafts/components/global_drafts_list/DraftSwipeActions/index.tsx

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useEffect, useRef} from 'react';
+import {useIntl} from 'react-intl';
+import {Animated, DeviceEventEmitter, TouchableWithoutFeedback} from 'react-native';
+import {Swipeable} from 'react-native-gesture-handler';
```

Comment:
There's a deprecation warning, should we use another component as suggested?

https://docs.swmansion.com/react-native-gesture-handler/docs/components/reanimated_swipeable

Reply:
Removed it and now using `ReanimatedSwipeable.